opencollab/arpack-ng
# Arpackmm

Do NOT merge this PR ! :D This is likely **not** yet working.

If this could go to a happy end, be aware that this PR is based on #155 : merging this PR implies merging #155.

Just want to share the idea (utility in connection with #149) , see if CI is OK (find eigen, test will be KO)... And get some help !

Currently: EXAMPLES/MATRIX_MARKET/arpackmm.sh fails for reason I can't figure out. 
@caliarim , @manavbhatia, anyone else: any help is welcome !...

To test this : 

```
~> git clone https://github.com/fghoussen/arpack-ng
~> git checkout arpackmm
~> sudo apt-get install libeigen3-dev
~> ./bootstrap; ./configure --enable-icb-exmm; make all check
~> tail -n 100 EXAMPLES/MATRIX_MARKET/test-suite.log
```

Problem: computed eigen vector is just wrong ! This is likely because the code "doesn't know" how to use the cubersome arpack lib !...
